src/holochainClient.js

```diff
@@ -111,9 +112,10 @@ export function conductorInstanceIdbyDnaAlias (instanceId) {
 }
 
 let holochainClient
+let hcFlag = false
 
-async function initAndGetHolochainClient () {
-  if (holochainClient) return holochainClient
+async function init () {
```

Comment:
on the fence about this, but I think I'd prefer this be called `initHolochainClient`.

It makes the specific difference between the two functions clearer.